jekyll/_cci2/non-aws.md

```diff
@@ -6,7 +6,7 @@ order: 1
 description: "Using CircleCI 2.0 static installation scripts."
 hide: true
 ---
-This article provides System Administrators with a preview of installing a clean CircleCI 2.0 installation if your existing 1.0 installation did not use AWS and instead used a custom hardware installation in the following sections:
+This article provides a System Administrators' overview of our preview 2.0 static installation process for non-AWS environments.
```

Comment:
- would moving "overview" closer to the verb be any clearer? There doesn't seem to be an easy way to do that without removing "System Administrator" (which may be fine!).
- change "our" to "CircleCI's" ([style link](https://github.com/circleci/circleci-docs/wiki/Style-Guide#referencing-circleci))

"This article provides an overview of CircleCI's preview 2.0 static installation process for non-AWS environments."